cicd/gitops/gitops-release-notes.adoc

```diff
@@ -0,0 +1,21 @@
+//OpenShift GitOps Release Notes
+include::modules/gitops-document-attributes.adoc[]
```

Comment:
I am not sure why we have this file in this PR. the other PR has this file and will be merged to both 4.7 and 4.8, so we actually do not need this file in this PR. You could use `git rm file_name` to remove this from the PR.